innopoints/views/variety.py

```diff
@@ -221,6 +223,30 @@ def purchase_variety(product_id, variety_id):
 
 # ----- StockChange -----
 
+@api.route('/stock_changes')
+@admin_required
+def list_purchases():
+    """List all of the purchases."""
+    default_limit = 24
+    default_page = 1
+
+    try:
+        limit = int(request.args.get('limit', default_limit))
```

Comment:
Why don't you pass `int` as the 3rd parameter to `args.get`?

Reply:
Well, in that case we won't catch and report the error, so both ways kind of work. I see none to be better than the other, and since this one allows proper error reporting, I think it's best